src/SourceBuild/patches/runtime/0001-Forward-the-pgoinstrument-argument-to-the-inner-buil.patch

```diff
@@ -0,0 +1,107 @@
+From 6d82f35999ffb5a1ed85931578e46d66816b385b Mon Sep 17 00:00:00 2001
+From: Jeremy Koritzinsky <jekoritz@microsoft.com>
+Date: Wed, 14 Feb 2024 11:43:33 -0800
+Subject: [PATCH] Forward the pgoinstrument argument to the inner build for
+ VMR/Source-build builds
+
+Build all assets when we're building PGO-instrumented assets in a vertical build/source-build scenario (where we won't have assets from a non-PGO-instrumented job available)
```

Comment:
Please add a `Backport: <#ISSUE/#PR>` comment per the [patching guidelines](https://github.com/dotnet/source-build/blob/main/Documentation/patching-guidelines.md#patch-guidelines).  This helps folks identify when to remove this patch in the dependency flow PRs.

Reply:
```suggestion
Build all assets when we're building PGO-instrumented assets in a vertical build/source-build scenario (where we won't have assets from a non-PGO-instrumented job available)

Backport: https://github.com/dotnet/runtime/pull/98446
```